activesupport/lib/active_support/error_reporter.rb

```diff
@@ -272,5 +277,32 @@ def ensure_backtrace(error)
 
         error.backtrace.shift(count)
       end
+
+      # Contains a stack of callables to run on an error's execution context before passing
+      # to subscribers. Allows creation of entries in error context that are shared by all
+      # subscribers.
+      #
+      # A context middleware receives the error and current state of the context hash.
+      # It must return a hash - the middleware stack returns the hash after it has
+      # run through all middlewares. A middleware can mutate or replace the hash.
+      class ErrorContextMiddlewareStack
```

Comment:
```suggestion
      class ErrorContextMiddlewareStack # :nodoc:
```

Since it's an implementation detail.